source/Octopus.Tentacle/Kubernetes/KubernetesScriptPodCreator.cs

```diff
@@ -234,6 +237,7 @@ void LogVerboseToBothLogs(string message, InMemoryTentacleScriptLog tentacleScri
 
         async Task<V1Container> CreateScriptContainer(StartKubernetesScriptCommandV1 command, IScriptWorkspace workspace, string podName, string scriptName, string homeDir)
         {
+            var spaceInformation = kubernetesPhysicalFileSystem.GetCachedStorageInformation();
```

Comment:
At this point, storage information should always be calculated so we can use the cached version. If it's not calculated, this method will fall back to calculating the storage information itself.